SPECS/ccache/ccache.spec

```
@@ -28,6 +28,11 @@ popd
 pushd build
 %make_install
 popd
+install -dm 755 $RPM_BUILD_ROOT%{_libdir}/ccache
+for n in cc gcc g++ c++ ; do
+    ln -sf ../../bin/ccache $RPM_BUILD_ROOT%{_libdir}/ccache/$n
+    ln -sf ../../bin/ccache $RPM_BUILD_ROOT%{_libdir}/ccache/%{_host}-$n
+done
```

Could you also run a linter against this spec? We're deprecating `$RPM_BUILD_ROOT` and the linter might also find other things anyway.
```suggestion
install -dm 755 %{buildroot}%{_libdir}/ccache
for n in cc gcc g++ c++ ; do
    ln -sf ../../bin/ccache %{buildroot}%{_libdir}/ccache/$n
    ln -sf ../../bin/ccache %{buildroot}%{_libdir}/ccache/%{_host}-$n
done
```

---

sure, will run the linter